charts/nexus-iq/values.yaml

```
@@ -295,3 +305,5 @@ configYaml:
         currentLogFilename: /var/log/nexus-iq-server/clm-server.log
         archivedLogFilenamePattern: /var/log/nexus-iq-server/clm-server-%d.log.gz
         archivedFileCount: 50
+
+extraManifests: []
```

(very minor we usually have the extra line at the end)
```suggestion
extraManifests: []

```